lib/git.js

```diff
@@ -444,10 +444,31 @@ class Git extends EventEmitter {
         callback = options;
         options = { type: 'http' };
       }
+      var enableCors = false;
```

Comment:
default should be in the options above, we should be checking in our code for options.cors on line 458.